tests/unit/ctags

```diff
@@ -1 +0,0 @@
-/usr/bin/ctags
```

Comment:
what is this?

Reply:
a mistake I made in the RandomMixin PR, thought I drop it now